hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2273,10 +2273,15 @@
             - Verify the correctness of keys provided to SRAM ctrl (main & ret), flash ctrl, keymgr,
               (note that keymgr does not have handshake), and OTBN.
 
-            X-ref'ed with IP tests that consume these signals.
+            X-ref'ed with the following IP tests that consume these signals:
+            - chip_sw_sram_scrambled_access
```

Comment:
IMO if missing, each of these tests should indicate ES SS is initialized by the startup code in the test ROM. They should also indicate that there are assertion checks at the output of all EDN channels to verify validity of returned data.

Reply:
thanks Sri for the suggestion. Added for sram and otbn testpoints.
Also file issue: https://github.com/lowRISC/opentitan/issues/14049 to write assertions for EDN output. I should finish writing these assertions soon.